organisaatio-service/src/main/java/fi/vm/sade/organisaatio/business/impl/OrganisaatioBusinessServiceImpl.java

```diff
@@ -748,7 +726,7 @@ private void updateOrganisaatioNameHierarchy(Organisaatio oppilaitos, Map<String
                             }
                         }
                         child.setPaivitysPvm(new Date());
-                        organisaatioRepository.save(child); // TODO works?
+                        child = organisaatioRepository.save(child); // TODO works?
```

Comment:
maybe remove the todo if this now works?

Reply:
removed Todo